docker/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM ubuntu:20.04
```

Comment:
We want it to be an ARG so we can swap between std ubuntu or ubuntu-nivida for access to GPU.

```
ARG BASE_IMAGE=ubuntu:20.04
FROM ${BASE_IMAGE}
```

Reply:
Fixed